config/notifiers.go

```diff
@@ -803,9 +802,8 @@ type MSTeamsConfig struct {
 	WebhookURL     *SecretURL                  `yaml:"webhook_url,omitempty" json:"webhook_url,omitempty"`
 	WebhookURLFile string                      `yaml:"webhook_url_file,omitempty" json:"webhook_url_file,omitempty"`
 
-	Title   string `yaml:"title,omitempty" json:"title,omitempty"`
-	Summary string `yaml:"summary,omitempty" json:"summary,omitempty"`
```

Comment:
I don't think we can remove this without a deprecation notice and subsequent deprecation period (for example, scheduled to be removed in the next 3 minor releases).

There are a number of projects that use Alertmanager such as [cortex](https://github.com/cortexproject/cortex) and [mimir](https://github.com/grafana/mimir), and removing this field will break Alertmanager configurations for tenants in those systems. The reason for this is Alertmanager uses `yaml.UnmarshalStrict`, which will fail if a configuration file contains a `summary` field.

Instead, we either need to keep it and find a use for it – or keep it, don't use it and document that it's deprecated so it can be removed at a later date.

Reply:
@grobinson-grafana 

Thanks for your review.

Good point. I'll keep it and document that it's deprecated.

Reply:
Hey @zhan9san @grobinson-grafana , are we going to have the feature released that msteams-power-platform anytime soon ?